.github/pull_request_template.md

```diff
@@ -1,15 +1,19 @@
+# PR Checklist
+
+- [ ] Ths PR is up-to-date with the top of all sub-component repositories except for those sub-components which are the subject of this PR. Please consult the ufs-weather-model [wiki](https://github.com/ufs-community/ufs-weather-model/wiki/Making-code-changes-in-the-UFS-weather-model-and-its-subcomponents) if you are unsure how to do this.
```

Comment:
Do we want to change "the top of all sub-component repositories" to "the versions of all sub-component repositories on the latest ufs-weather-model develop branch"?  This way I think developers will not be confused with the latest version of each sub component repository.

Reply:
One question. Aren't those sub-components which are the subject of this PR need also be up-to-date with the top of their respective repositories? I thought 'up-to-date with the top of ...' means pulling in and merging the top rather than being at the top.